src/main/resources/ApplicationContext.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For such filename ordinary start from the small letter.
ApplicationContext.xml -> applicationContext.xml 
It is better to use java code to configure context.